blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/IOConfig.java

```diff
@@ -0,0 +1,62 @@
+package com.rackspacecloud.blueflood.io;
+
+import com.rackspacecloud.blueflood.service.Configuration;
+import com.rackspacecloud.blueflood.service.CoreConfig;
+
+import java.net.InetSocketAddress;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Created by shin4590 on 3/24/16.
+ */
+public class IOConfig {
+
+    protected static final Configuration config = Configuration.getInstance();
```

Comment:
Since IOConfig is also a singleton, dont think it matters much whether we leave it as static or non-static. I will say this though. I like less static. Spring has spoiled me.